rayon-core/src/configuration.rs

```diff
@@ -63,7 +63,8 @@ impl Configuration {
     /// Set a closure which takes a thread index and returns
     /// the thread's name.
     pub fn set_thread_name<F>(mut self, closure: F) -> Self
-    where F: FnMut(usize) -> String + 'static {
+        where F: FnMut(usize) -> String + 'static
```

Comment:
I was running a brand new update of rustfmt so I'm surprised if it didn't come through. Have the new changes made it to release?